vars/prod-vars.yaml

```diff
@@ -199,6 +199,69 @@ clusters:
           613431313731633132613335383865303566
         tlsClientConfig:
           insecure: true
+  - name: cluster-dh-apps.ocp.prod.psi.redhat.com
+    data:
+      name: apps.ocp.prod.psi.redhat.com
```

Comment:
Can we change this name to have a dh prefix?

Reply:
added